scripts/obtain/dotnet-install.sh

```diff
@@ -583,7 +584,7 @@ downloadcurl() {
 
     local failed=false
     if [ -z "$out_path" ]; then
-        curl --retry 10 -sSL -f --create-dirs $remote_path || failed=true
+        curl --retry 10 -sSL -f --create-dirs -o $remote_path || failed=true
```

Comment:
I don't think this is right. -o $remore_path will try to output of $remote_path. 
As far as I could tell, curl path was always right as curl would output to stdout by default.